src/index.js

```diff
@@ -31,7 +30,5 @@ const Root = () => (
 );
 
 
-ReactDOM.render(
-  <Root />,
-  document.getElementById('root')
-);
+const root = createRoot(document.getElementById('root'));
```

Comment:
Just curious, what kind of warning happened here?

Reply:
Please refer to [this](https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis).